pkgs/applications/science/electronics/geda/default.nix

```diff
@@ -12,6 +12,10 @@ stdenv.mkDerivation rec {
   configureFlags = "--disable-update-xdg-database";
   buildInputs = [ pkgconfig guile gtk flex gawk ];
 
+  patchPhase = ''
+    sed -i -e 's/\#!\/usr\/bin\/perl -w/\#!\/usr\/bin\/env perl\nuse warnings;\n;/' ./utils/scripts/refdes_renum
```

Comment:
i can make that change, but patch-shebangs.sh wont pass on the warning line i take it?  so i will still have to do sed to turn back on the use warning flag right?